### PR TITLE
Provide a minimal Gradle script to run Gluecodium

### PR DESCRIPTION
**Pull request by DanielKamkha**
Resolves: #163
Signed-off-by: Daniel Kamkha <daniel.kamkha@here.com>

### CODE REVIEW

File: gluecodium/src/main/java/com/here/gluecodium/cli/OptionReader.kt [hunk 1]
```diff
@@ -176,10 +180,8 @@ object OptionReader {
     }
 
     fun printUsage() {
-        val header = "Gluecodium - Generate APIs from LimeIDL files\n\n"
-        val footer = "\nPlease report issues at /dev/null"
```
Comment (Dschoordsch) [inline gluecodium/src/main/java/com/here/gluecodium/cli/OptionReader.kt]: Could have been replaced with a link to the actual issue tracker

File: tools/README.md [hunk 2]
```diff
@@ -8,3 +8,15 @@ IDE (or any other IDE derived from it, e.g. Android Studio, etc.).
...
+_What does it do_: provides a minimal set of scripts to run Gluecodium from a binary distribution,
+without a need of any compilation or download.
+
+_How to install_: copy the "launcher" directory locally.
+
+_How to run_: from command line, while in the local "launcher" directory, execute `gradle run`. To
+pass command line parameters to Gluecodium, execute `gradle run --args="..."`. For example, to see
+the version number of Gluecodium you're running, execute `gradle run --args="-version"`. To run a
+non-latest version of Gluecodium, execute `gradle run -Pversion=N.N.N`, for example
+`gradle run -Pversion=6.3.0`.
```
Comment (Dschoordsch) [inline tools/README.md]: +1 I would modify it to be `gradle run -Pversion=6.3.0 --args="-help"`

File: gluecodium/src/main/java/com/here/gluecodium/cli/OptionReader.kt (hunk 1 above)
Comment (DanielKamkha) [reply gluecodium/src/main/java/com/here/gluecodium/cli/OptionReader.kt]: Done

File: tools/README.md (hunk 2 above)
Comment (DanielKamkha) [reply tools/README.md]: Done

**Pull request merged**